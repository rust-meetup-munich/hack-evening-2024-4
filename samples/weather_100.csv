San Salvador;28.5
Willemstad;33.6
New Delhi;36.9
Chișinău;17.7
Chittagong;33.3
Tucson;13.8
Whitehorse;5.2
Albuquerque;23.0
Ouagadougou;28.9
Portland (OR);29.8
Conakry;41.8
Entebbe;12.8
Rangpur;40.7
Rabat;20.0
N'Djamena;16.6
Kuopio;-1.8
Houston;30.9
Dikson;-11.2
Oulu;0.3
Dakar;39.6
Palermo;13.1
Chongqing;15.0
Hanga Roa;24.3
Kingston;26.8
St. Louis;22.7
Minsk;19.7
Tegucigalpa;22.5
Manama;38.1
Lhasa;24.1
Vientiane;12.1
Tegucigalpa;30.1
Edmonton;15.4
Pyongyang;6.1
Ouahigouya;23.7
Rome;14.8
Bridgetown;28.6
Cracow;4.6
Mahajanga;30.4
Lubumbashi;19.7
Sapporo;4.9
Kinshasa;17.1
Cairns;27.2
Lubumbashi;21.5
Ouagadougou;33.4
Louisville;20.4
Lagos;16.2
Frankfurt;28.5
Istanbul;17.5
Asmara;10.7
Boise;23.7
Praia;8.6
Yellowknife;7.6
Wellington;19.3
Minneapolis;17.2
Addis Ababa;22.5
Saskatoon;1.5
Bergen;-10.8
Mombasa;28.3
New Orleans;9.5
Oulu;-18.9
Niigata;24.4
Bosaso;3.5
Portland (OR);7.6
Kyoto;16.3
Tamanrasset;10.8
Bridgetown;30.2
London;6.3
Yaoundé;26.5
Da Nang;39.0
Stockholm;-2.0
Erzurum;-7.2
Brisbane;18.2
Wrocław;19.2
Kyiv;-2.9
Ottawa;29.9
Surabaya;50.5
Tegucigalpa;31.6
Lviv;19.4
Karonga;23.7
Ghanzi;0.2
Pyongyang;-4.4
Tashkent;4.6
Kuwait City;19.7
Christchurch;7.0
Pyongyang;-12.0
Kansas City;-8.1
Ghanzi;33.5
Surabaya;40.9
Port Sudan;35.9
Bishkek;9.8
Managua;43.2
Anadyr;-1.2
Sofia;16.2
Sarajevo;11.5
Wau;23.2
San Salvador;24.2
Astana;-1.4
Bangui;19.0
St. John's;14.0
Canberra;10.8
