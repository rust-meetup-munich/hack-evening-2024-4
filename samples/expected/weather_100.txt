{
    San Salvador=24.2/26.4/28.5,
    Willemstad=33.6/33.6/33.6,
    New Delhi=36.9/36.9/36.9,
    Chișinău=17.7/17.7/17.7,
    Chittagong=33.3/33.3/33.3,
    Tucson=13.8/13.8/13.8,
    Whitehorse=5.2/5.2/5.2,
    Albuquerque=23.0/23.0/23.0,
    Ouagadougou=28.9/31.1/33.4,
    Portland (OR)=7.6/18.7/29.8,
    Conakry=41.8/41.8/41.8,
    Entebbe=12.8/12.8/12.8,
    Rangpur=40.7/40.7/40.7,
    Rabat=20.0/20.0/20.0,
    N'Djamena=16.6/16.6/16.6,
    Kuopio=-1.8/-1.8/-1.8,
    Houston=30.9/30.9/30.9,
    Dikson=-11.2/-11.2/-11.2,
    Oulu=-18.9/-9.3/0.3,
    Dakar=39.6/39.6/39.6,
    Palermo=13.1/13.1/13.1,
    Chongqing=15.0/15.0/15.0,
    Hanga Roa=24.3/24.3/24.3,
    Kingston=26.8/26.8/26.8,
    St. Louis=22.7/22.7/22.7,
    Minsk=19.7/19.7/19.7,
    Tegucigalpa=22.5/28.1/31.6,
    Manama=38.1/38.1/38.1,
    Lhasa=24.1/24.1/24.1,
    Vientiane=12.1/12.1/12.1,
    Edmonton=15.4/15.4/15.4,
    Pyongyang=-12.0/-3.4/6.1,
    Ouahigouya=23.7/23.7/23.7,
    Rome=14.8/14.8/14.8,
    Bridgetown=28.6/29.4/30.2,
    Cracow=4.6/4.6/4.6,
    Mahajanga=30.4/30.4/30.4,
    Lubumbashi=19.7/20.6/21.5,
    Sapporo=4.9/4.9/4.9,
    Kinshasa=17.1/17.1/17.1,
    Cairns=27.2/27.2/27.2,
    Louisville=20.4/20.4/20.4,
    Lagos=16.2/16.2/16.2,
    Frankfurt=28.5/28.5/28.5,
    Istanbul=17.5/17.5/17.5,
    Asmara=10.7/10.7/10.7,
    Boise=23.7/23.7/23.7,
    Praia=8.6/8.6/8.6,
    Yellowknife=7.6/7.6/7.6,
    Wellington=19.3/19.3/19.3,
    Minneapolis=17.2/17.2/17.2,
    Addis Ababa=22.5/22.5/22.5,
    Saskatoon=1.5/1.5/1.5,
    Bergen=-10.8/-10.8/-10.8,
    Mombasa=28.3/28.3/28.3,
    New Orleans=9.5/9.5/9.5,
    Niigata=24.4/24.4/24.4,
    Bosaso=3.5/3.5/3.5,
    Kyoto=16.3/16.3/16.3,
    Tamanrasset=10.8/10.8/10.8,
    London=6.3/6.3/6.3,
    Yaoundé=26.5/26.5/26.5,
    Da Nang=39.0/39.0/39.0,
    Stockholm=-2.0/-2.0/-2.0,
    Erzurum=-7.2/-7.2/-7.2,
    Brisbane=18.2/18.2/18.2,
    Wrocław=19.2/19.2/19.2,
    Kyiv=-2.9/-2.9/-2.9,
    Ottawa=29.9/29.9/29.9,
    Surabaya=40.9/45.7/50.5,
    Lviv=19.4/19.4/19.4,
    Karonga=23.7/23.7/23.7,
    Ghanzi=0.2/16.9/33.5,
    Tashkent=4.6/4.6/4.6,
    Kuwait City=19.7/19.7/19.7,
    Christchurch=7.0/7.0/7.0,
    Kansas City=-8.1/-8.1/-8.1,
    Port Sudan=35.9/35.9/35.9,
    Bishkek=9.8/9.8/9.8,
    Managua=43.2/43.2/43.2,
    Anadyr=-1.2/-1.2/-1.2,
    Sofia=16.2/16.2/16.2,
    Sarajevo=11.5/11.5/11.5,
    Wau=23.2/23.2/23.2,
    Astana=-1.4/-1.4/-1.4,
    Bangui=19.0/19.0/19.0,
    St. John's=14.0/14.0/14.0,
    Canberra=10.8/10.8/10.8
}
